spec/views/orders/new.html.slim_spec.rb

```diff
@@ -8,8 +8,6 @@
 
   it 'form contain select box with product category' do
     render
-    Category.find_each do |c|
-      expect(rendered).to match /#{sanitize c.name}/
-    end
+    expect(rendered).to match /#{sanitize Category.all.sample.name}/
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.